UnityProject/Assets/Scripts/Items/Stackable.cs

```diff
@@ -306,15 +308,21 @@ public bool WillInteract(InventoryApply interaction, NetworkSide side)
 	public void ServerPerformInteraction(InventoryApply interaction)
 	{
 		//clicking on it with an empty hand when stack is in another hand to take one from it
-		if (interaction.IsFromHandSlot && interaction.IsToHandSlot && interaction.FromSlot.IsEmpty)
+		if (interaction.IsFromHandSlot && interaction.IsToHandSlot && interaction.FromSlot.IsEmpty && !interaction.IsAltClick)
 		{
 			//spawn a new one and put it into the from slot with a stack size of 1
 			var single = Spawn.ServerPrefab(prefab).GameObject;
+			Logger.Log(single.ToString());
 			if (single == null || single.GetComponent<Stackable>() == null) return;
 			single.GetComponent<Stackable>().SyncAmount(amount, 1);
 			Inventory.ServerAdd(single, interaction.FromSlot);
 			ServerConsume(1);
 		}
+		//Alt+clicking with empty hand calls splitting menu UI.
+		else if(interaction.IsFromHandSlot && interaction.IsToHandSlot && interaction.FromSlot.IsEmpty && interaction.IsAltClick)
+		{
+			UIManager.Instance.SplittingMenu.Enable();
+		}
```

Comment:
This is server side so won't work for clients, think theres a interface for clientside only interactions

Reply:
It works and I saw application of same command in other script. Idk how to replace it with something else

Reply:
> 

It works in the editor as there you are a client and server, but this wont work on headless

Reply:
`public bool WillInteract(InventoryApply interaction, NetworkSide side)` In here add it

Reply:
```
                //Alt+clicking with empty hand calls splitting menu UI.
		if(interaction.IsAltClick)
		{
			UIManager.Instance.SplittingMenu.Enable();
                        return false;
		}
```

Reply:
Also want to check `side.Client` is true

Reply:
Fixed in new commit